presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergUtil.java

```diff
@@ -243,4 +250,33 @@ public static Map<String, String> createIcebergViewProperties(ConnectorSession s
                 .put(TABLE_TYPE_PROP, ICEBERG_TABLE_TYPE_VALUE)
                 .build();
     }
+
+    public static int parseVersion(String metadataLocation)
```

Comment:
Can we create a small unit test that asserts behavior based off the Iceberg spec?

Reply:
Great idea! I'll add to my test list